utilities/test_suite/rpp_test_suite_common.h

```diff
@@ -59,6 +74,22 @@ inline T validate_pixel_range(T pixel)
     return pixel;
 }
 
+// replicates the last image in a batch to fill the remaining images in a batch
+void replicate_last_image_to_fill_batch(const string& lastFilePath, vector<string>& imageNamesPath, vector<string>& imageNames, const string& lastFileName, int noOfImages, int batchCount)
+{
+    std::string filePath = lastFilePath;
+    std::string fileName = lastFileName;
+    if (noOfImages > 0 && noOfImages < batchCount)
+    {
+        for (int i = noOfImages; i < batchCount; i++)
+        {
+            imageNamesPath.push_back(filePath);
+            imageNames.push_back(fileName);
+        }
+    }
+}
+
+// Retrieve the interpolation type used for image resizing or scaling operations.
```

Comment:
Instead of retrieve, mention as returns
Applicable to all such instances in this file

Reply:
Done